logstage/logstage-config/src/main/scala/com/github/pshirshov/izumi/logstage/config/codecs/RenderingPolicyCodec.scala

```diff
@@ -0,0 +1,102 @@
+package com.github.pshirshov.izumi.logstage.config.codecs
+
+import com.github.pshirshov.izumi.distage.config.codec.ConfigReader
+import com.github.pshirshov.izumi.distage.model.reflection.universe.RuntimeDIUniverse.u
+import com.github.pshirshov.izumi.logstage.api.rendering.RenderingPolicy
+import com.github.pshirshov.izumi.logstage.config.codecs.RenderingPolicyCodec.{NamedRenderingPolicy, RenderingPolicyMapper}
+import com.typesafe.config.{Config, ConfigFactory, ConfigValue}
+import RenderingPolicyCodec._
+
+import scala.util.Try
+
+class RenderingPolicyCodec(policyMappers: Set[RenderingPolicyMapper[_ <: RenderingPolicy]]) extends ConfigReader[RenderingPolicy] {
+  private val policyInstancesMappers = scala.collection.mutable.HashMap.empty[String, NamedRenderingPolicy]
+  private val policyMappersMem = scala.collection.mutable.HashMap.empty[String, Config => _ <: RenderingPolicy]
```

Comment:
Why not `private val policyMappersMem = mutable.HashMap[String, RenderingPolicy](policyMappers.toSeq.map(m => m.path.toString -> m): _*)` ?